### PR TITLE
Koby izunaso GitHub miscellany

### LINKED ISSUE #99: Update Debug Lab
Debugging lab is well organized but lacks so more details. For example, there are other ways to add a breakpoint in eclipse not mentioned. I would like to add more concrete text and images to improve the context of debugging lab. I would also like to add more tips to debugging as coming into it as a first timer can be overwhelming without prior advice and knowledge. Adding more tips and links to other articles on how to approach debugging could help.

### PR DESCRIPTION
**Pull request by Kobyizunaso**
Fixes #99 
Add text with more debugging tips
Added link to detailed list of dubbing tips'
Added another way of selecting breakpoint besides the one already
Added image on how to select breakpoint

### CODE REVIEW

**Review by onzfonz — CHANGES_REQUESTED** [review]
I would remove the letters from the headings
If you want it can be another list
Also rename you breakpoint file as it has a space between the end of the name and the extension

Lastly, i would like you to break up the paragraph on lines 100 to 118 and any other text not as a hard wrap but to break it up by clauses and sentences
Any text after a period or comma typically goes on the next line down.
Please see most of the file to see how it should be formatted.
Anything that is too long to fit on one line should be broken down so that each line has a phrase that is meaningful
This makes it easier to edit in the future in meaningful ways.

**Comment by onzfonz** [comment]
you also need to link to the issue correctly as the canvas assignment states @Kobyizunaso and we mention in discord

**Comment by Kobyizunaso** [comment]
Updated changes you suggested and updated issue commentary.

**Review by Kobyizunaso — COMMENTED** [review]
Removed alphabets. Spaces new tip properly and spaces images for breakpoint.

**Review by onzfonz — CHANGES_REQUESTED** [review]
Please use the ***add suggestions to batch*** button from the files changed tab to commit the suggestions as a single commit and fix the remaining issues that I've pointed out here.

image

image

image

Also again, please link the issue and use this picture to help you edit the fixes portion in the description correctly

File: docs/labs/3-Debug.md [hunk 1]
```diff
@@ -94,6 +94,24 @@ take a minute to override ```toString()```
 so you can get a faster way to view important details about an object,
 ```toString``` works both in the console and in the debugger.
 
+### Finding the joy in Debugging
```
Comment (onzfonz) [inline docs/labs/3-Debug.md:97]:
```suggestion
### Finding the Joy in Debugging
```

File: docs/labs/3-Debug.md [hunk 2]
```diff
@@ -94,6 +94,24 @@ take a minute to override ```toString()```
...
+The developers who excel at fixing problems in their code see debugging
+as a puzzle to solve and actually like finding and fixing errors more
+than writing new code.
+
+Debugging is basically about solving problems. 
+It involves figuring out what's wrong,
+understanding how different parts of the code work together,
+and making everything work smoothly again. Just like any challenge,
+debugging has its difficulties, but when you successfully fix the issues,
+it feels really rewarding.
```
Comment (onzfonz) [inline docs/labs/3-Debug.md:108]:
```suggestion
and making everything work smoothly again.
Just like any challenge, debugging has its difficulties,
but when you successfully fix the issues, it feels really rewarding.
```

File: docs/labs/3-Debug.md [hunk 3]
```diff
@@ -94,6 +94,24 @@ take a minute to override ```toString()```
...
+
+It involves thinking creatively and having a good overall understanding of the code.
+Every time developers debug, they have a chance to learn something new.
```
Comment (onzfonz) [inline docs/labs/3-Debug.md:111]:
```suggestion
Every time you debug, *you have a chance to learn something new*.
```

File: docs/labs/3-Debug.md [hunk 4]
```diff
@@ -94,6 +94,24 @@ take a minute to override ```toString()```
...
+It helps them understand the system better, discover new tools,
```
Comment (onzfonz) [inline docs/labs/3-Debug.md:112]:
```suggestion
Debugging helps you understand the system better, discover new tools,
```

File: docs/labs/3-Debug.md [hunk 5]
```diff
@@ -94,6 +94,24 @@ take a minute to override ```toString()```
...
+and improve their problem-solving skills.
```
Comment (onzfonz) [inline docs/labs/3-Debug.md:113]:
```suggestion
and improve your problem-solving skills.
```

**Review by onzfonz — CHANGES_REQUESTED** [review]
Please do not commit the suggestions unless the add suggestions to batch is working for you.
If you are stuck, please send a screenshot and refrain from committing or repeating the same steps.

**Comment by onzfonz** [comment]
@Kobyizunaso hope the conference is going well!  Let me know if you have any questions about following the steps outlined here so that you can be approved.

File: docs/labs/3-Debug.md [hunk 6]
```diff
@@ -94,6 +94,24 @@ take a minute to override ```toString()```
 so you can get a faster way to view important details about an object,
 ```toString``` works both in the console and in the debugger.
 
+### Finding the Joy in Debugging
+
+The developers who excel at fixing problems in their code see debugging
+as a puzzle to solve and actually like finding and fixing errors more
+than writing new code.
+
+Debugging is basically about solving problems.
```
Comment (onzfonz) [inline docs/labs/3-Debug.md:103]:
```suggestion
Debugging is basically about solving problems.
```

**Review by onzfonz — CHANGES_REQUESTED** [review]
please commit this

**Review by onzfonz — APPROVED** [review]
LGTM

**Pull request merged**